examples/default.yaml

```diff
@@ -297,6 +297,7 @@ audio:
   # QEMU audiodev, e.g., "none", "coreaudio", "pa", "alsa", "oss".
   # VZ driver, use "vz" as device name
   # Choosing "none" will mute the audio output, and not play any sound.
```

Comment:
I know this is not part of this PR, but reading the docs here makes me wonder what the difference is between `""` and `"none"` beyond that `"none"` doesn't work with VZ. Why would you use `"none"` instead of just leaving off the audio device?

Reply:
Actually some distributions don't work properly without an audio card, so the default ("") is to add one but leave it disconnected to any speakers. It can be explicitly removed ("none"), if you know that the OS will still boot...

Reply:
Technically, "" omits the qemu flag

Reply:
Yes, but this is end-user documentation. We should not punt by telling them to read the qemu docs and the Lima sources, but explain what each valid choice does, and why they would want to pick `""` over `"none"` or vice versa. Or just drop `"none"` from our comments if there is no reason a user would ever want that.

Reply:
The defaults changed again QEMU 8.2, so if we want to document _what_ the default drivers are - it needs some more research and testing.